OracleOGG/Oracle/12.2.0.1.1/Dockerfile

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Since: June, 2017
+# Author: hadi.koesnodihardjo@oracle.com
+# Description: script to build a Docker image for Oracle GoldenGate for Oracle.
+#
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+#
+FROM oraclelinux:7-slim
+MAINTAINER OGG PM <bobby.curtis@oracle.com>
+#Container Additional Environment Variables
+ENV OGG_SHIPHOME=V100692-01.zip \
+    OGG_HOME=/u01/app/oracle/product/12.2.0.1/oggcore_1 \
+    OGG_PORT=9500 \
+    ORACLE_HOME=/u01/app/oracle/product/12.1.0.2/dbhome_1 \
+    ORACLE_BASE=/u01/app/oracle \
+    ORACLE_SID=db12c \
+    ORACLE_INVENTORY=/u01/app/oraInventory \
+    ORACLE_DB_LIST_PORT=1521 \
+    DB_CHARSET=WE8ISO8859P9 \
+    SYS_PASSWORD=Welcome1 \
```

Comment:
We do not allow hard-coded passwords. You need generate the password automatically at runtime as well as allow the end-user to provide a password at runtime using `-e`.